src/main/plugins/dictionary-plugin/dictionary-plugin.ts

```diff
@@ -74,10 +74,6 @@ export class DictionaryPlugin implements ExecutionPlugin {
             const keys = Object.keys(definition.meaning);
             keys.forEach((key) => {
                 definition.meaning[key]
-                    .filter((entry: any) => {
-                        return entry.definition
-                            && entry.example;
-                    })
                     .forEach((entry: any) => {
                         dictionaryResults.push({
                             definition: entry.definition,
```

Comment:
I agree on removing the check for the example, because we don't need it anyway. So I suggest you also remove the example from the interface here:

https://github.com/oliverschwendener/ueli/blob/fe882ca8803896f1133146cdb58f40367b281c06/src/main/plugins/dictionary-plugin/dictionary-plugin.ts#L12-L17

and here:

https://github.com/oliverschwendener/ueli/blob/fe882ca8803896f1133146cdb58f40367b281c06/src/main/plugins/dictionary-plugin/dictionary-plugin.ts#L82-L87

But why did you remove the check for the definition? If we removed this filter function totally we would get dictionary results with empty definitions, which are required here:

https://github.com/oliverschwendener/ueli/blob/fe882ca8803896f1133146cdb58f40367b281c06/src/main/plugins/dictionary-plugin/dictionary-plugin.ts#L92-L102

Reply:
Yeah you're right, It never occurred to me that the api would send empty definition field, so it safe to keep the filter for definition only.